test/testdata/deployednettemplates/recipes/scenario1/node.json

```diff
@@ -5,18 +5,18 @@
     "TelemetryURI": "{{TelemetryURI}}",
     "EnableMetrics": false,
     "MetricsURI": "{{MetricsURI}}",
-    "ConfigJSONOverride": "{  \"TxPoolExponentialIncreaseFactor\": 1, \"DNSBootstrapID\": \"<network>.algodev.network\", \"DeadlockDetection\": -1, \"PeerPingPeriodSeconds\": 30, \"BaseLoggerDebugLevel\": 4, \"EnableProfiler\": true }",
-   "AltConfigs": [
-    {
-        "APIToken": "{{APIToken}}",
-        "EnableBlockStats": true,
-        "EnableTelemetry": true,
-        "TelemetryURI": "{{TelemetryURI}}",
-        "EnableMetrics": true,
-        "MetricsURI": "{{MetricsURI}}",
-        "ConfigJSONOverride": "{  \"TxPoolExponentialIncreaseFactor\": 1, \"DNSBootstrapID\": \"<network>.algodev.network\", \"DeadlockDetection\": -1, \"PeerPingPeriodSeconds\": 30, \"EnableAgreementReporting\": true, \"EnableAgreementTimeMetrics\": true, \"EnableAssembleStats\": true, \"EnableProcessBlockStats\": true, \"BaseLoggerDebugLevel\": 4, \"EnableProfiler\": true }",
-        "FractionApply": 0.2
-    }
+    "ConfigJSONOverride": "{  \"TxPoolExponentialIncreaseFactor\": 1, \"DNSBootstrapID\": \"<network>.algodev.network\", \"DeadlockDetection\": -1, \"PeerPingPeriodSeconds\": 30, \"BaseLoggerDebugLevel\": 4, \"CadaverSizeTarget\": 0  }",
```

Comment:
is there any specific reason you removed the `EnableProfiler` flag ?

Reply:
I removed it because it wasn't in the net.json file, i'll add it back